detox/android/build.gradle

```diff
@@ -3,9 +3,9 @@ buildscript {
         isOfficialDetoxLib = true
         kotlinVersion = '1.2.0'
         dokkaVersion = '0.9.18'
-        buildToolsVersion = '29.0.3'
-        compileSdkVersion = 29
-        targetSdkVersion = 29
+        buildToolsVersion = '30.0.2'
```

Comment:
I know we talked about this but seems like I need you to refresh my memory... why not use build tools 31.x.x?

Reply:
build tools will fail unless a user is on gradle 7 toolchain (gradle and android gradle plugin) which I don't think is a guarantee https://stackoverflow.com/questions/68387270/android-studio-error-installed-build-tools-revision-31-0-0-is-corrupted (there's a google issuetracker bug as well, they appear to have no intention of fixing it after months). 30 is thus more compatible for now, surely

I have personally spent my time on this, total waste ;-) https://github.com/invertase/notifee/issues/211

Reply:
Understood. Thanks!